community/neo4j/src/test/java/schema/IndexPopulationIT.java

```diff
@@ -159,16 +159,13 @@ public void shutdownDatabaseDuringIndexPopulations()
 
     private void prePopulateDatabase( GraphDatabaseService database, Label testLabel, String propertyName )
     {
-        for ( int j = 0; j < 100; j++ )
+        for ( int j = 0; j < 10_000; j++ )
         {
-            for ( int i = 0; i < 100; i++ )
+            try ( Transaction transaction = database.beginTx() )
```

Comment:
I think batching here would be a good idea. Or just create all of the 10000 nodes in one transaction. Do as you please with this.

Reply:
i think it does not matter